app/app.js

```diff
@@ -0,0 +1,56 @@
+import {App, IonicApp, Platform} from 'ionic/ionic';
+
+import {HelloIonicPage} from './pages/hello-ionic/hello-ionic';
+import {ListPage} from './pages/list/list';
+
+@App({
+  templateUrl: 'build/app.html'
+})
+class MyApp {
+  constructor(app: IonicApp, platform: Platform) {
+
+    // set up our app
+    this.app = app;
+    this.platform = platform;
```

Comment:
We also define some data in the constructor which will make it available throughout the rest of the class and in the view. We create a reference to the app and platform components that were passed in, and we also create some data describing the pages in our application.